.env.example

```
@@ -25,4 +25,8 @@ NEXT_PUBLIC_APP_URL=
 
 # Google Maps
 NEXT_PUBLIC_GOOGLE_MAPS_API_KEY=
-GOOGLE_MAPS_API_KEY=
\ No newline at end of file
+GOOGLE_MAPS_API_KEY=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GOOGLE_MAPS_API_KEY key should go before the NEXT_PUBLIC_GOOGLE_MAPS_API_KEY key

(UnorderedKey)